README.md

```diff
@@ -4,73 +4,114 @@ Vocab is a strongly typed internationalization framework for React.
+This is the language Vocab will assume when it sees a `translation.json` file without a language prefix.
 
 ```js
+// vocab.config.js
 module.exports = {
-  devLanguage: 'en',
   languages: [{ name: 'en' }, { name: 'fr' }]
+  devLanguage: 'en',
 };
 ```
 
+See the [configuration] section for more configuration options.
+
+[IETF language tags]: https://en.wikipedia.org/wiki/IETF_language_tag
+[configuration]: #configuration
+
 ### Step 3: Set the language using the React Provider
 
-Vocab doesn't tell you how to select or change your language. You just need to tell Vocab what language to use.
+Vocab uses React's context API to provide information for your translation lookups.
+To tell Vocab which language to use, wrap your app in a `VocabProvider` component and pass in a `language` prop corresponding to one of the language names configured in your `vocab.config.js` file.
 
-**Note:** Using methods discussed later we'll make sure the first language is loaded on page load. However, after this, changing languages may then lead to a period of no translations as Vocab downloads the new language's translations.
+> [!NOTE]
+> Using methods discussed later we'll make sure the first language is loaded on page load.
+> However, after this, changing languages may lead to a period of no translations as Vocab downloads the new language's translations.
+
+```tsx
+// src/App.tsx
+
+import { VocabProvider } from '@vocab/react';
+
+const App = ({ children }) => {
+  return (
+    <VocabProvider language="en">{children}</VocabProvider>
+  );
+};
+```
 
-**src/App.tsx**
+If you need to customize the locale for your language, you can pass a `locale` prop to the `VocabProvider` component.
+This tells Vocab which locale to use when formatting your translations.
 
 ```tsx
+// src/App.tsx
+
 import { VocabProvider } from '@vocab/react';
 
 function App({ children }) {
   return (
-    <VocabProvider language={language}>
+    <VocabProvider language="myCustomLanguage" locale="en">
       {children}
     </VocabProvider>
   );
 }
 ```
 
-### Step 4: Create initial values and use them
+See [here][overriding the locale] for more information on how and when to use the `locale` prop.
 
-A translation file is a JSON file consisting of a flat structure of keys, each with a message and an optional description.
+[overriding the locale]: #overriding-the-locale
 
-**Note:** Currently, to create a new translation it must be placed inside a folder ending in **`.vocab`**, this folder suffix can be configured with the `translationsDirectorySuffix` configuration value.
+### Step 4: Create translations
 
-**`./example.vocab/translations.json`**
+A translation file is a JSON file consisting of a flat structure of keys.
+Each key must contain a `message` property, and optionally a `description` property.
+
+Rather than creating one giant file for each language's translations, Vocab enables you to locate translations as close as possible to the components that use them.
```

Comment:
Rephrasing the second half, may be subjective

```suggestion
Rather than creating one giant file for each language's translations, Vocab enables you to co-locate the translations alongside their consuming components.
```